pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -471,21 +478,6 @@ func readInstanceID(searchOrder string) (string, error) {
 
 // check opts for OpenStack
 func checkOpenStackOpts(openstackOpts *OpenStack) error {
-	lbOpts := openstackOpts.lbOpts
-
-	// if need to create health monitor for Neutron LB,
```

Comment:
Just to be sure, neutron LB still works if we remove this right? We are still supporting neutron LB , but just not the default option right

Reply:
The check here is removed because the default value is specified. It doesn't affect neutron-lbaas.